ButterSample/ButterSample/SampleEndpoints/SampleEndpointB.swift

```diff
@@ -5,18 +5,36 @@
 import Foundation
 import Butter
 
+struct SampleRequestBodyB: Encodable {
+	let first = "jason"
+	let last = "derulo"
+	let occupation = "unknown"
+}
+
 struct SampleEndpointB: Endpoint {
     var scheme: String { "http" }
     
     var host: String { "localhost" }
     
     var port: Int? { 8070 }
     
-    var path: String? { "/test/b" }
+    var path: String? { "bungheads/apply/" }
```

Comment:
where is the `/` between the host and path being added? did it change in this diff and i'm not seeing it

Reply:
will look into this